data/map planets.txt

```diff
@@ -3301,6 +3346,10 @@ planet "New Argentina"
 	description `New Argentina is a world that is ideal for ranching but suitable for little else. Ranchers live as individual families or small communities surrounded by hectares of grazing land for cattle. As a result, the population is still quite low here, and the locals live isolated lives, with most families living many kilometers away from the nearest school or hospital or even from their nearest neighbor.`
 	description `	Cattle handlers here do their work on horseback, because they believe modern vehicles are more likely to spook the animals.`
 	spaceport `The spaceport is on the outskirts of one of New Argentina's largest towns... which has a population of barely ten thousand people. Many of the dock workers are teenagers or barely past their teens, children of ranchers who are not drawn to the solitary sort of life that their parents enjoy. Aside from the church and the hospital, the largest building in town is "The Grange," a function hall with a sign out front advertising square dances and line dances three nights a week.`
+	spaceport `	The Grange is currently set up for Remembrance Day celebrations. The population inside consists mostly of the older residents of the planet, many of them likely having traveled tens of thousands of kilometers to arrive here.`
```

Comment:
Does ES use kilometers or miles?

Reply:
Generally kilometers from what I can tell -- I found dozens of references to kilometers in human news/missions/campaign, but only one use of miles for a line of news.